libraries/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -414,6 +422,54 @@ export class WorkspaceInstallManager extends BaseInstallManager {
 
     // TODO: Remove when "rush link" and "rush unlink" are deprecated
     LastLinkFlagFactory.getCommonTempFlag(this.rushConfiguration).create();
+
+    // Stage 2 rebuild pending
+    if (this.twoStageInstall && !this.options.ignoreScripts) {
+      const rebuildArgs: string[] = ['rebuild', '--pending', '--recursive'];
+
+      if (
```

Comment:
It feels like a lot of this is duplicate from the method we use to append the default parameters. Maybe these can be made into a common method?